src/Components/Results/Programs/ProgramCard.tsx

```diff
@@ -2,38 +2,75 @@ import { Link, useParams } from 'react-router-dom';
 import { Program } from '../../../Types/Results';
 import { FormattedMessage } from 'react-intl';
 import ResultsTranslate from '../Translate/Translate';
+import { useWindowSize } from '@uidotdev/usehooks';
```

Comment:
Hi @Mdemenko1 - thanks for the work on this. The updates look great. Only note - as a best practice, we'd like to keep the amount of packages we use to a minimum since this can increase the chances of something breaking outside of our control, causing issues that are hard to debug and creating some additional work to keep all of the packages up to date. Since we can use the built-in 'window' object to get the screen width, let's use that instead to keep our packages use to a minimum where we can. [This StackOverflow Post](https://stackoverflow.com/a/36862446) provides more details on how to use the `window` object in React. Let me know if you have additional questions on this. Happy to provide more clarity if needed.